src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/EventTracePipelineUnitTests.cs

```diff
@@ -47,7 +47,6 @@ await using (var testExecution = StartTraceeProcess("TraceStopTest"))
 
                 await using var pipeline = new EventTracePipeline(client, settings, async (s, token) =>
                 {
-                    //The buffer must be read in order to not hang. The Stop message will not be processed otherwise.
                     await s.CopyToAsync(buffer);
```

Comment:
Maybe add the dispose stream check to this test too so that we verify it is disposed when the pipeline is stopped.